File: 핸드폰 번호 가리기.py
"""
def solution(phone_number):
    
    abc = []
    adc = ""
    abc = phone_number

    for i in range(len(phone_number)-4):
       adc += '*'
    
    return (adc+phone_number[-4:])
"""
#2022.11.12
def solution(phone_number):
    return '*'*len(phone_number[:-4])+phone_number[-4:]

File: 핸드폰 번호 가리기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>
#include <string.h>

char* solution(const char* phone_number) {
    char* answer = (char*)malloc(sizeof(char) * (strlen(phone_number) + 1));
    strcpy(answer, phone_number);
    
    for (int i = 0; i < strlen(answer)-4; i++){
        answer[i] = '*';
    }
    return answer;
}
